va/va.go

```diff
@@ -673,49 +685,158 @@ func (va *ValidationAuthorityImpl) performRemoteValidation(ctx context.Context,
 					va.log.Errf("Remote VA %q.PerformValidation failed: %s", rva.Addresses, err)
 				}
 			}
-			errors <- err
-		}(remoteVA)
+			va.log.Infof("remote VA %d returned %#v", index, err)
```

Comment:
This looks like a leftover debugging log line? If the intent is to keep this we should add some context like the authz id / hostname being validated.

Reply:
Good catch, I had a lot of those at one point (;﹏;)

Fixed in 6cf6b1a